src/mo_frontend/typing.ml

```diff
@@ -1419,7 +1460,7 @@ and check_exp' env0 t exp : T.typ =
   | TupE exps, T.Tup ts when List.length exps = List.length ts ->
     List.iter2 (check_exp env) ts exps;
     t
-  | ObjE exp_fields, T.Obj(T.Object, fts) ->
+  | ObjE (exp_fields, []), T.Obj(T.Object, fts) -> (* FIXME: bases? *)
```

Comment:
Can we offer better error messages than the standard `M0096` (below) if the bases are known?